elasticlib/Elastic.java

```diff
@@ -1,4 +1,4 @@
-package frc.robot.util;
+package Elasticlib;
```

Comment:
What's the reason for changing the package name?

Reply:
No real reason, I just liked it better